packages/components/src/internal/components/editable/EditableGridPanelDeprecated.tsx

```diff
@@ -47,13 +49,21 @@ interface Props extends EditableGridProps {
     getIdField?: (tabId?: number) => string;
     getTabTitle?: (tabId?: number) => string;
     getTabHeader?: (tabId?: number) => ReactNode;
+    initialEmptyRowCount?: number;
+    onCellModify?: () => void;
+    onRowCountChange?: (rowCount?: number) => any;
 }
 
 interface State {
     activeTab: number;
+    rowsInitialized: boolean;
 }
 
 export class EditableGridPanelDeprecated extends ReactN.Component<Props, State, GlobalAppState> {
+    static defaultProps = {
```

Comment:
with these changes in the `EditableGridPanelDeprecated` component, should we bump the other non-LKB modules that use this component so that we get TC coverage of these changes? (or maybe you already have those FBs and I haven't see the PRs yet)

Reply:
I'm basically tackling them one at a time since I have to drink from the firehose every time. I have SM and Biologics passing. Next up is FM. I am not concerned about any other usages.